src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries.mdx

```diff
@@ -47,7 +41,17 @@ When you run a NRQL query, it displays the number of data points inspected, as s
 
 In this context, the "events" term is used in a general sense to refer to all [NRQL-available data points](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql#what-you-can-query) that are inspected in a query.
 
-Every New Relic account has limits on the total number of data points that can be inspected in a given time range. These limits vary depending on which [data option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) you have: 
+The following features and actions count towards the inspected count limit for your New Relic account, and are impacted when the limit is reached. When the count drops below the limit in the next time period, any restrictions are removed.
```

Comment:
@zuluecho9 features in themselves don't generate the cost, right? It's the "action" of using them, right? Could you simplify this by just referring to "actions"?